src/TrueLayer/Payments/Model/Beneficiary.cs

```diff
@@ -43,6 +44,11 @@ public MerchantAccount(string merchantAccountId)
             /// Gets or inits A reference for the payment. Not visible to the remitter.
             /// </summary>
             public string? Reference { get; init; }
+
+            /// <summary>
+            /// Gets or inits verification remitter information
```

Comment:
See other comment about the scope of the verification type

Reply:
I have changed it. However, [in the official API ref](https://docs.truelayer.com/reference/create-payment#:~:text=required-,MERCHANT,-ACCOUNT), they mention the remitter in the description 
<img width="545" alt="image" src="https://github.com/user-attachments/assets/72510893-5ff2-4514-bdca-646c62554880">